src/org/omegat/Bundle.properties

```diff
@@ -1158,10 +1152,6 @@ LFC_ERROR_FILE_DOESNT_EXIST=File {0} does not exist!
 
 PM_ERROR_SAVE=Preferences system save operation failed!
 
-PM_WARNING_PARSEERROR_ON_READ=Parse error encountered reading preferences file!
-
-PM_UNSUPPORTED_ENCODING=Unsupported file encoding
```

Comment:
Last modified in 377d7485682d1f2f75b48dc70e742090a712a387  in 2006 which was used in Preferences.java long time ago.